### PR TITLE
Add planetary.org password rules

### PR DESCRIPTION
**Pull request by aaronraimist**
### Overall Checklist
- [x] I agree to the project's [Developer Certificate of Origin](https://github.com/apple/password-manager-resources/blob/main/DEVELOPER_CERTIFICATE_OF_ORIGIN.md)
- [x] The top-level JSON objects are sorted alphabetically
- [x] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update

#### for password-rules.json
- [x] The given rule isn't particularly standard and obvious for password managers
- [x] Generated passwords have been tested from this rule using the [Password Rules Validation Tool](https://developer.apple.com/password-rules/)
- [x] Information has been included about the website's requirements (eg. screenshots, error messages, steps during experimentation, etc.)
- [x] The PR isn't documenting something that would be a common practice among password managers (e.g. minimal length of 6)

Screen Shot 2020-07-12 at 12 16 36 PM

### CODE REVIEW

**Comment by rmondello** [comment]
Hi @aaronraimist! This website accepts Safari’s generated password format, which includes a `-`, so this rule is incomplete.

File: quirks/password-rules.json [hunk 1]
```diff
@@ -267,6 +267,9 @@
     "pixnet.cc": {
         "password-rules": "minlength: 4; maxlength: 16; allowed: lower, upper;"
     },
+    "planetary.org": {
+        "password-rules": "required: lower; required: upper; required: digit; minlength: 5; maxlength: 20;"
```
Comment (rmondello) [inline quirks/password-rules.json:271]: Let’s make this comprehensive in terms of allowed symbols.

Comment (aaronraimist) [reply quirks/password-rules.json:271]: How do I use the `allowed` key? The validation tool doesn't seem to like `allowed: ascii-printable;`

Screen Shot 2020-07-13 at 8 23 08 PM

Comment (rmondello) [reply quirks/password-rules.json:271]: This was a bug in the tool, which we just pushed a fix for. Try again now!

**Comment by rmondello** [comment]
Thanks @aaronraimist!

**Pull request merged**